app/views/sessions/new.html.erb

```diff
@@ -1,15 +1,26 @@
-<h1>Login</h1>
-
-<%= form_tag sessions_path do |form| %>
-  <div class="field">
-    <%= label_tag :email %>
-    <%= text_field_tag :email %>
-  </div>
-  <div class="field">
-    <%= label_tag :password %>
-    <%= password_field_tag :password %>
+<section class="terms-header bg-light px-3 py-3 pt-md-5 pb-md-4 mx-auto">
```

Comment:
on success label does not seem green 

![image](https://user-images.githubusercontent.com/8007722/71911872-852ad200-3142-11ea-9f3a-1cc18b0a578a.png)

Reply:
should be fixed now